pkgs/development/ocaml-modules/ocaml-gettext/camomile.nix

```diff
@@ -0,0 +1,16 @@
+{ buildDunePackage, ocaml_gettext, camomile, ounit }:
+
+buildDunePackage {
+  pname = "gettext-camomile";
+  inherit (ocaml_gettext) src version useDune2;
+
+  propagatedBuildInputs = [ camomile ocaml_gettext ];
+
+  doCheck = true;
+  checkInputs = [ ounit ];
+
+  meta = ocaml_gettext.meta // {
+    description = "Internationalization library using camomile (i18n)";
+  };
+
+}
```

Comment:
```suggestion
  };
}
```

Reply:
Can you please explain why it is better not to put a blank line here so that I don’t repeat the same mistake in the future? A link to formatting guidelines would be appreciated.

Reply:
There is none currently and not even the proposed nixpkgs-fmt would reformat this.

Reply:
> Can you please explain why it is better not to put a blank line here so that I don’t repeat the same mistake in the future?

Because no blank line is needed there.

> A link to formatting guidelines would be appreciated.

There are none and people would bikesheed until death about it.

> proposed nixpkgs-fmt would reformat this.

because it is not designed to do this.